packages/mystmd-py/setup.cfg

```diff
@@ -25,6 +25,9 @@ keywords = markdown, latex, writing-software, scientific-visualization, pdf-gene
 packages = find:
 python_requires = >=3.6
 include_package_data = True
+install_requires =
+    jupyter-server
+    ipykernel
```

Comment:
Can we hold off on these changes until it is in the CLI?

Reply:
I think if this comes out @agoose77, we can go ahead with a merge and then start working on the CLI pr for real.